microsoft-community-training/analytics/custom-reports/power-bi-integration.md

```diff
@@ -57,10 +59,49 @@ In this article, you will learn more about connecting Power BI with the Azure SQ
     :::image type="content" source="../../media/image%28366%29.png" alt-text="Select the tables to import and click on load button":::
 
 6. Once you hit on load all your selected tables will be imported into Power Bi right pane and available under fields.
+
     :::image type="content" source="../../media/image%28367%29.png" alt-text="Loaded tables will be imported to PowerBI":::
 
 7. Use the database schema from [this document](database-schema.md) to build relations required for your custom reporting requirements
 
->[!Tip]
->To get started, MCT provides customers with custom visualizations through Power BI Dashboard to effectively track learner’s progress across courses.
->[**MCT PowerBI custom Dashboard.zip**](https://github.com/MicrosoftDocs/microsoft-community-training/files/6967550/MCT.PowerBI.custom.Dashboard.zip) this folder contains custom PowerBI dashboard and a file with steps to replace the existing data source with your instance's database to view out of the box insights from your data through custom created visualizations.
+## Using custom Power BI dashboard
+
+To get started, MCT provides customers with a custom made PowerBI dashboard with interactive visualizations to effectively track learner’s progress across courses.
```

Comment:
To help you get started on creating your own PowerBI reports here are some samples. These sample reports with their interactive visualizations can be used to create further customized reports for your specific requirements. 

The pbix files for the sample reports can be downloaded from here -